modules/extensions/src/mask/mask.js

```diff
@@ -1,5 +1,6 @@
 import {LayerExtension, log} from '@deck.gl/core';
 import mask from './shader-module';
+import {projectPosition} from '@deck.gl/core/shaderlib/project/project-functions';
```

Comment:
You can't import functions like this. When published the files will be in dist/es5 or dist/esm. This only works in test because we set module aliases in the test harness.

This function is available under the exported `project` shader module.

Reply:
I couldn't find where it was exported. I added it to the core module exports